ivy/functional/frontends/torch/tensor.py

```diff
@@ -774,6 +774,12 @@ def is_cuda(self):
     def is_meta(self):
         return "meta" in ivy.dev(self.ivy_array)
 
+    @with_unsupported_dtypes(
+        {"2.1.0 and below": ("uint16", "bfloat16", "bool")}, "torch"
+    )
+    def positive(self, *, out=None):
```

Comment:
```suggestion
    def positive(self):
```
This function doesn't take the 'out' kwarg (https://pytorch.org/docs/stable/generated/torch.Tensor.positive.html#torch.Tensor.positive)